Formula/osrm-backend.rb

```diff
@@ -3,6 +3,7 @@ class OsrmBackend < Formula
   homepage "http://project-osrm.org/"
   url "https://github.com/Project-OSRM/osrm-backend/archive/v5.4.3.tar.gz"
   sha256 "501b9302d4ae622f04305debacd2f59941409c6345056ebb272779ac375f874d"
+  head "https://github.com/Project-OSRM/osrm-backend.git", :using => :git
```

Comment:
Argh, I meant _un_necessary so: we agree. Can you remove it please @kkdd? Thanks!